### DISCUSSION TITLE
We need testers!

### DISCUSSION THREAD

**Discussion by marticliment** [Devolutions/UniGetUI#1022] [Announcements]
We need testers who are willing to test the first beta of WingetUI 2.0.0
While aesthetically this version has not changed a lot, the main codebase has been rewritten. More details on the release:
https://github.com/marticliment/WingetUI/releases/tag/2.0.0-beta

Thanks!

**Comment by ppvnf** [comment]
Performance seems much better indeed!

**Reply by DailenG** [reply]
I'll be testing it out today

**Comment by marticliment** [comment]
A second beta has been released with more changes: https://github.com/marticliment/WingetUI/releases/tag/2.0.0-beta2

**Comment by ppvnf** [comment]
Hey, I am excited for the new version. Do you plan to release another beta version?

**Reply by marticliment** [reply]
soon, hopefully this weekend.

I want to get some things fixed before releasing a new beta

**Comment by marticliment** [comment]
Third beta of WingetUI 2.0.0 had been released. It includes pip and npm!

**Reply by Spacellary** [reply]
My pip and npm packages updated quickly and with no issues I could notice 👍

The console output is great

I'm actually very happy since even as not a dev per se I use some handy npm tools (eg: webtorrent-cli)

**Comment by Spacellary** [comment]
2.0.0 beta 3 is so polished, I love it.
